response.go

```diff
@@ -101,3 +103,25 @@ func (r *HtmlResponse) Render() error {
 
 	return nil
 }
+
+type StreamResponse struct {
+	code     int
+	instance *gin.Context
+	writer   func(w contractshttp.StreamWriter) error
+}
+
+func (r *StreamResponse) Render() error {
+	r.instance.Status(r.code)
+
+	w := NewStreamWriter(r.instance)
+
+	clientGone := r.instance.Writer.CloseNotify()
```

Comment:
I noticed the method is deprecated.

<img width="1093" alt="image" src="https://github.com/user-attachments/assets/1ddff1d5-cd13-4ab4-bcec-c36d59acdfd2">

We should use:

```
ctx := r.instance.Request.Context()
...
case <-ctx.Done():
```

Reply:
Ohh, you are right, missed it.